app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -17,6 +19,11 @@ class Form526DocumentUploadFailureEmail < Job
         timestamp = Time.now.utc
         form526_submission_id, supporting_evidence_attachment_guid = msg['args']
 
+        log_info = { job_id:, timestamp:, form526_submission_id:, error_class:, error_message:,
```

Comment:
Was there a reason for moving this log to before the form 526 job status? I don't think it matters either way just curious

Reply:
My thinking was that early logging would bring value should a failure occur later in the block.